ironpython-git/PKGBUILD

```diff
@@ -22,8 +20,10 @@ pkgver() {
 
 build() {
   cd "$srcdir/$pkgname"
-  sed -i "/NoWarn/s|444;||" Solutions/Common.proj
-  xbuild Solutions/IronPython.sln /p:Mono=true /p:Configuration="Release"
+  export LC_ALL="en_US.UTF-8"
```

Comment:
Is this (and `LANG=`) required? User environments should be respected where possible, so if we can do it without forcing a US locale that would be great